src/graphql/main/schema.graphql

```diff
@@ -725,9 +725,10 @@ input OnChainPaymentSendInput {
   walletId: WalletId!
 }
 
-type OnChainTxFee {
-  amount: SatAmount!
-  targetConfirmations: TargetConfirmations!
+type OnChainTxFeePayload {
```

Comment:
Is this breaking? Have we discussed how to deal with this change in the mobile repo?

Reply:
Fixed here: https://github.com/GaloyMoney/galoy/pull/1408/commits/be1c84a34c7abd3b553128b38e1bdb75f7a4dca6